doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-new-project-creating-proc.adoc

```diff
@@ -15,14 +15,10 @@ image::getting-started/spaces-teams.png[]
 
 . Click *Add Project*.
 . Enter `traffic-violation` in the *Name* field.
-. Click *Configure Advanced Options* to expand the GAV fields and enter the following values:
-* *Group ID*: `org.kie.example.traffic`
-* *Artifact ID*: `traffic-violation`
-* *Version*: `1.0.0`
```

Comment:
Due to this change wee need to update two more places where we guide users to pick **org.kie.example.traffic.traffic_violation** package. Such package will not exist anymore. Users need to pick **com.myspace.traffic_violation** package.

Reply:
Great catch Jozef. Sorry that I missed this. Updating accordingly.